libs/driver/shopify/src/provider.ts

```diff
@@ -7,4 +21,16 @@ import { provideShopifyApolloDriver } from './graphql/provide-shopify-apollo-dri
  * @param accessToken - The Shopify Storefront API access token.
  * @returns An Apollo client provider configured with a Shopify store domain and Storefront API access token.
  */
-export const provideShopifyDriver = (domain: string, accessToken: string) => provideShopifyApolloDriver(domain, accessToken);
+export const provideShopifyDriver = (config: DaffShopifyDriverConfig | InjectionToken<DaffShopifyDriverConfig> ) => provideNamedApollo(() => {
+  const resolvedConfig = config instanceof InjectionToken ? inject(config) : config;
```

Comment:
Where does the config injection token come from? Shouldn't we provide one (and mandate its usage)?